frontend/packages/topology/src/components/page/TopologyPage.tsx

```diff
@@ -100,6 +101,15 @@ export const TopologyPage: React.FC<TopologyPageProps> = ({
   const viewType =
     (queryParams.get('view') as TopologyViewType) || topologyViewState || defaultViewType;
 
+  React.useEffect(() => {
+    const lastOverviewOpen = JSON.parse(
+      sessionStorage.getItem(LAST_TOPOLOGY_OVERVIEW_OPEN_STORAGE_KEY) ?? '{}',
+    );
+    if (loaded && namespace in lastOverviewOpen) {
+      setQueryArgument('selectId', lastOverviewOpen[namespace]);
+    }
+  });
```

Comment:
Please add these deps here. With this two the hook was only called once. Without them it was called whenever I select a topology node.

```suggestion
  }, [loaded, namespace]);
```

Reply:
Updated it.